lib/pubsub/topic.js

```diff
@@ -83,59 +100,55 @@ Topic.formatName_ = function(projectId, name) {
  *
  * @throws {Error} If no message is provided.
  *
- * @param {*} message - The message to publish.
+ * @param {object|object[]} message - The message(s) to publish.
+ * @param {*} message.data - The contents of the message in any format.
```

Comment:
Actually, I'll just say "The contents of the message." - saying the type is done from the `*`.